src/lib/components/filters/content.svelte

```diff
@@ -8,41 +8,69 @@
         InputSelectCheckbox,
         InputDateTime
     } from '$lib/elements/forms';
-    import { createEventDispatcher, onMount } from 'svelte';
-    import { operators, addFilter, queries, type TagValue } from './store';
+    import { onMount, createEventDispatcher } from 'svelte';
+    import { operators, addFilter, queries, tags } from './store';
     import type { Column } from '$lib/helpers/types';
     import type { Writable } from 'svelte/store';
     import { TagList } from '.';
     import { Icon, Layout } from '@appwrite.io/pink-svelte';
     import { IconPlus } from '@appwrite.io/pink-icons-svelte';
 
-    // We cast to any to not cause type errors in the input components
-    /* eslint  @typescript-eslint/no-explicit-any: 'off' */
-    export let value: any = null;
-    export let columns: Writable<Column[]>;
-    export let columnId: string | null = null;
-    export let arrayValues: string[] = [];
-    export let operatorKey: string | null = null;
-    export let singleCondition = false;
+    let {
+        value = $bindable(null),
+        columns,
+        columnId = $bindable(null),
+        arrayValues = $bindable([]),
+        operatorKey = $bindable(null),
+        singleCondition = false
+    }: {
+        value?: string | number | string[] | null;
```

Comment:
revert the type and the comments for this prop since strict values will cause binding issues.